bindata/network/multus/multus.yaml

```diff
@@ -116,29 +116,109 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: multus
-      initContainers:
-      - name: multus-binary-copy
+      containers:
+      - name: kube-multus
         image: {{.MultusImage}}
-        command: ["/entrypoint/cnibincopy.sh"]
+        command: [ "/bin/bash", "-c", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh;
```

Comment:
you still need `set -e`, or you can add `-ec` to the second command arg.  Otherwise if cnibincopy fails your scirpt keeps going.

Reply:
I changed to run this commands with `&&` and added `set -e` to the cni binary copy script.

Reply:
set -e means you don't need && :)